packages/ui/src/base/tokens.scss

```diff
@@ -270,5 +270,5 @@ body {
   // SHADOW
   --fs-shadow                          : none;
   --fs-shadow-darker                   : 0 0 10px rgb(0 0 0 / 20%);
-  --fs-shadow-hover                    : 0 0 0 1px var(--fs-color-main-4);
+  --fs-shadow-hover                    : 0 1px 4px rgb(0 0 0 / 10%), 0 6px 8px rgb(0 0 0 / 10%);
```

Comment:
why are we changing this value here?

Reply:
It is because the current `--fs-shadow-hover` have a strange border aspect and with this configuration we have a better "shadow" aspect